### PR TITLE
Resolved status mixed-case issue, expanded log messaging, removed unn…

### PR DESCRIPTION
**Pull request by TonyB9000**
…ecessary exit

Another set of eyes ... and developing good habits ...

### CODE REVIEW

File: warehouse/warehouse/scripts/move_to_publication.py [hunk 1]
```diff
@@ -156,8 +156,7 @@ def main():
                 outstream.write(message)
                 con_message("info", message)
         else:
-            con_message("error", f"cannot obtain message_file (from message_path) from environment for message {message}")
-            sys.exit(1)
+            con_message("info", f"cannot obtain message_file (from message_path) from environment for message {message}")
```
Comment (tomvothecoder) [inline warehouse/warehouse/scripts/move_to_publication.py:159]: Does `con_message()` support `"warning"`?

**Review by tomvothecoder — APPROVED** [review]
Looks good Tony 👍 

Quick tip: You can link a related GH issue(s) to the PR to provide context behind the motive 
for the changes. To do that, you just need to type **#ISSUE** (e.g., #78) in the PR description. There is also an option in the sidebar called "Linked Issues" if you prefer that way.

File: warehouse/warehouse/scripts/move_to_publication.py (hunk 1 above)
Comment (TonyB9000) [reply warehouse/warehouse/scripts/move_to_publication.py]: Thanks Tom.  I'd been toggling "info", and "debug" for so long I'd forgotten "warning".

**Comment by tomvothecoder** [comment]
No problem. If a PR is more ad-hoc and there isn't an existing issue, you can add a 1-2 sentence description in the PR or retroactively create the issue + link it.

**Review by chengzhuzhang — APPROVED** [review]

**Pull request merged**